docs/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md

```diff
@@ -16,6 +16,10 @@ import TabItem from "@theme/TabItem";
 
 [Web Modeler](../../components/modeler/about-modeler.md) serves as a powerful tool for the development and deployment of processes and process applications. While Web Modeler simplifies one-click deployment for development, professional teams often rely on continuous integration and continuous deployment (CI/CD) pipelines for automated production deployments. The [Web Modeler API](/apis-tools/web-modeler-api/index.md) facilitates integration of Web Modeler into these pipelines, aligning with team practices and organizational process governance.
 
+- For low-risk processes, you can use the Web Modeler [process application development pipeline](/docs/components/modeler/web-modeler/process-application-pipeline.md) to quickly develop and progress process application releases through the stages of a standard development life cycle. [Milestone comparison](/docs/components/modeler/web-modeler/milestones.md#compare-milestones) (Visual and XML diffing) and [Git Sync](/docs/components/modeler/web-modeler/git-sync.md) provide a powerful combination.
```

Comment:
This needs some conceptual language cleanup, but it's a good summary. Should I start using life cycle rather than lifecycle in the rest of my communications?

Reply:
No, stick with lifecycle for now, I'll change the docs - we've just discussed this in DevEx and I think we should stick with lifecycle (one word) in these docs.